gradle.properties

```diff
@@ -3,20 +3,20 @@
 
 pluginGroup = com.github.madwareru.intellijronremix
 pluginName = RON Extended Support
-pluginVersion = 0.3.1
+pluginVersion = 0.4.0
```

Comment:
Please change the version to 0.3.2 since the change is minor and does not seem to bring many changes

Reply:
updated